utils.py

```diff
@@ -143,7 +154,7 @@ def _setup_full_prompts_to_test_on(input_fields_list, regex_key_idx_list, select
 
     full_prompt_string_list = []
     for input_element, idx in zip(inputs, selected_dataset_ids):
-        full_prompt_string_list.append(input_element if n_shot == 0 else demonstration_string + "\n\n" + input_element)
+        full_prompt_string_list.append(demonstration_string + "\n\n" + input_element)
```

Comment:
I think even for 0-shot, the instruction (demonstration_definition) should still be there.